src/tox/execute/local_sub_process/read_via_thread_windows.py

```diff
@@ -1,21 +1,25 @@
 """
 On Windows we use overlapped mechanism, borrowing it from asyncio (but without the event loop).
 """
+from __future__ import annotations
+
 import logging  # pragma: win32 cover
 from asyncio.windows_utils import BUFSIZE  # pragma: win32 cover
 from time import sleep  # pragma: win32 cover
-from typing import Callable, Optional  # pragma: win32 cover
+from typing import Callable
 
 import _overlapped  # type: ignore[import]  # pragma: win32 cover
 
 from .read_via_thread import ReadViaThread  # pragma: win32 cover
 
+# mypy: warn-unused-ignores=false
```

Comment:
What is the reason we need this now but not when supporting 3.6?

Reply:
Partially unrelated. The rewrite of files caused some unused ignore to show up. So I moved the global do not warn unusued ignores to file level. We need this settings in this file because it's the only way to make mypy pass on windows and linux too (as some ignores are windows only or linux only).